<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswordDialog</class>
 <widget class="QWidget" name="ChangePasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="oldPasswordEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPasswordEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="submitNewPasswordButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLabel" name="newPasswordLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>新密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPasswordEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="oldPasswordLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>旧密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirmPasswordLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确认新密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
